src/pages/Home/styles.ts

```diff
@@ -1 +1,7 @@
-export default 'ss';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+    conatiner: { marginTop: 30 },
```

Comment:
use theme to add margins and padding, like: marginTop: theme.spacing(3)

Reply:
okay